lib/seek/samples/attribute_type_handlers/cv_list_attribute_type_handler.rb

```diff
@@ -0,0 +1,29 @@
+module Seek
+  module Samples
+    module AttributeTypeHandlers
+      class CVListAttributeTypeHandler < BaseAttributeHandler
+        class MissingControlledVocabularyException < AttributeHandlerException; end
```

Comment:
this exception already exists for `CVAttributeTypeHandler`. Infact I noticed another duplicate of `MissingLinkedSampleTypeException` (not related to this PR).

Not a blocker for this PR but perhaps a future refactoring task to bring them all under `AttributeTypeHandlers::`